ts/components/styles/wallet.ts

```diff
@@ -0,0 +1,192 @@
+/**
+ * Common styles used across the components
+ */
+
+import { StyleSheet } from "react-native";
+
+import color from "color";
+import variables from "../../theme/variables";
+
+// TODO: when all the required screens are added,
+// this style will be refactored to get rid of useless
+// styles and aggregate repeated ones to improve
+// readability and maintenability
+// @https://www.pivotaltracker.com/story/show/157769361
+export const WalletStyles = StyleSheet.create({
```

Comment:
You are using `fontFamily: variables.fontFamily` to style text. Are you sure without this it doesn't work? If it is a problem of the base theme we must fix it.

Reply:
It works because what you see in ts/themes/components/Text.ts get merged (and overrides) the base Text theme of native-base in this case `https://github.com/GeekyAnts/NativeBase/blob/master/src/theme/components/Text.js`.